global_localization_engine: BBS
globalmap_downsample_resolution: 0.25
query_downsample_resolution: 0.25
